docs/03-running.md

```diff
@@ -0,0 +1,29 @@
+# Running
+
+Media Atom Maker requires Node 6.X, so ensure you're running that version, or run `nvm use` first.
+
+## (legacy) Angular App
+To run the Angular App:
+
+- Generate javascript files by running `gulp watch`
+- Start sbt by running `./sbt app/run`
+- Open the app by going [here](https://video.local.dev-gutools.co.uk/atoms)
+
+## React App
+There are two ways to run the React App.
+
+Once its running, open the app by going [here](https://video.local.dev-gutools.co.uk/videos)
```

Comment:
I'd make this sentence the first of the section, then the instructions for two ways of running the app would come right after the stating that there are two ways of running the app.